zombie-bite-scripts/helpers.ts

```diff
@@ -42,15 +44,16 @@ export async function monitMigrationFinish(
   base_path?: string,
   rc_port?: string | number,
   ah_port?: string | number,
+  logger?: Logger,
```

Comment:
The logger is generally made an globally exported variable that you can import everywhere, no need to pass it functions.